lib/rules/jsx-closing-bracket-location.js

```diff
@@ -19,13 +19,14 @@ const messages = {
 
 module.exports = {
   meta: {
+    type: 'layout',
     docs: {
       description: 'Validate closing bracket location in JSX',
       category: 'Stylistic Issues',
       recommended: false,
       url: docsUrl('jsx-closing-bracket-location'),
     },
-    fixable: 'code',
+    fixable: 'whitespace',
```

Comment:
why is this change correct?

(if it is correct, i'd like to land it separately, before this PR)

Reply:
it was part of the original PR

I'm 100% not sure what this property means because the documentation doesn't mention what is the difference between `code` and `whitespace`; possibly `whitepsace` means that a fix will only change whitespace characters

Reply:
@ljharb I've reverted the changes to `fixable` and `category` in made by the previous PR author as they are not related to `--fix-type` (purpose of this PR); please have a look, thanks!

Reply:
looks like this change was correct, because the fix changes only spacing before the bracket (indents + newline), see https://github.com/eslint/eslint/discussions/15489#discussioncomment-1915826

Reply:
@sryze can we make a separate PR that changes all the "code"s to "whitespace"s?